### ISSUE TITLE
Coherent Sheaf type changing

### ISSUE THREAD

**Issue by thelongdiv** [Macaulay2/M2#227]
Any Coherent sheaf F has the following property in M2: F^**0 is of type sheaf of rings.  This is a problem for many reasons, one of which is O_X tensor F isn't a possible operation.  F^**0(0) is of type coherent sheaf, however.  So F^**1 ** F^**-1 is of type Coherent Sheaf.

Note on the local side of things, if I is of type ideal, I^0 is still of type ideal (1)

**Comment by DanGrayson** [comment]
Hmm, when I try ```F^0``` I get an error:

```

i1 : X = Proj (QQ[x,y,z])

o1 = X

o1 : ProjectiveVariety

i2 : F = OO_X^1

        1
o2 = OO
       X

o2 : coherent sheaf on X, free

i3 : F^0
stdio:3:2:(3): error: expected more than 0 arguments

i4 : F^3

        3
o4 = OO
       X

o4 : coherent sheaf on X, free
```

**Comment by thelongdiv** [comment]
Hi Daniel,

For tensor powers, F^**0, this is problematic.

Andrew

On Fri, Apr 24, 2015 at 5:36 PM, Daniel R. Grayson <notifications@github.com

**Comment by DanGrayson** [comment]
Did you intend to write F^**0 in your original report?  If so, that's certainly a bug: any
tensor power of a sheaf should be a sheaf.  Fixed in commit 38233c5d3e707dc96bb5c779b9686a07f5a8f5fc.

**Comment by thelongdiv** [comment]
Yes I did.  I edited the comment.  Thank you for your help.

Andrew

On Fri, Apr 24, 2015 at 5:40 PM, Daniel R. Grayson <notifications@github.com
